Also write GNU-stack note on dummy files used for stub libraries.
--- a/tools/winebuild/import.c
+++ b/tools/winebuild/import.c
@@ -1675,2 +1675,3 @@
         output( "\t.text\n" );
+        output_gnu_stack_note();
     }
